bash_functions.sh

```diff
@@ -181,6 +181,12 @@ setup_php_env() {
 
     echo "Added ENV to php:"
     grep -E '(VIRTUAL_HOST|ServerIP|PHP_ERROR_LOG)' "$PHP_ENV_CONFIG"
+
+    if [[ -n ${PHP_ENV_CGI_MEMORY_LIMIT} ]]; then 
+        grep -qP 'memory_limit = 128M' || \
```

Comment:
What if it changes here sometimes? Wouldn't it be better to place an additional config file under php directory with `.ini` extension with desired settings? Like proposed [here](https://github.com/pi-hole/docker-pi-hole/issues/595#issuecomment-699701827)

Reply:
Nice hack, It will probably fix the issue for sure. Thank to bring it however I tend to think that a bug should not be fixed by the enduser with a config file mounted by a custom volume

I would hope to get an Env var to modify at the container creation.

I am involved in another community project (NethServer) and this kind of modification are done with a simply esmith property. We try to provide a solution for the most common usages without issues for the system administrator